command/config.go

```diff
@@ -0,0 +1,87 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+
+	httpclient "github.com/filecoin-project/storetheindex/api/v0/admin/client/http"
+	"github.com/urfave/cli/v2"
+)
+
+var ConfigCmd = &cli.Command{
+	Name:  "config",
+	Usage: "Dynamically modifies and shows the current configuration",
```

Comment:
Looks like this pertains to the log levels config.  Is this comment correct, because it seems to say that it can modify any part of the config.

Reply:
I envisioned eventually the `config` command does that. For now it just modifies the log level.

Happy to change that description to be less aspirational